src/structure/widget.ts

```diff
@@ -465,7 +578,11 @@ export class MoleculeViewer {
         };
     }
 
-    private _atomSelected(atom: Partial<$3Dmol.AtomSpec>): void {
+    /**
+     * Func
+     * @param atom
+     */
+    private _selectAtom(atom: Partial<$3Dmol.AtomSpec>): void {
         // use atom.serial instead of atom.index to ensure we are getting the
         // id of the atom inside the central cell when using a supercell
```

Comment:
Is there a reason for using "atom.serial" instead of "guid"? Thinking of consistency of naming.

Reply:
That's the naming & data format used by 3Dmol